app/test_main.py

```diff
@@ -1 +1,39 @@
 # write your code here
+import pytest
+from unittest.mock import patch
+from app.main import can_access_google_page
+
+
+url = "https://www.google.com"
+
+
+@pytest.mark.parametrize(
+    "test_url,valid_google_url,has_internet_connection,expected",
+    [
+        (url, True, True, "Accessible"),
```

Comment:
here we can don't have to pass url as a parameter since it's the same everywhere. You can pass it directly to test. It would make sense to parametrize it if it was different to every data set